src/ensembl/src/shared/components/external-link/ExternalLink.tsx

```diff
@@ -37,15 +37,20 @@ const ExternalLink = (props: ExternalLinkProps) => {
 
   return (
     <span className={styles.container}>
-      <LinkIcon className={iconClass} />
-      <a
-        className={linkClass}
-        href={props.to}
-        target="_blank"
-        rel="nofollow noreferrer"
-      >
-        {props.linkText}
-      </a>
+      {props.to && (
```

Comment:
This goes back to our old conversation from some PRs back :-) I don't think, from the semantic standpoint, it makes sense to have an `ExternalLink` component that doesn't have a `props.to`. If there is not `props.to`, then it's not a link; so we shouldn't use the `ExternalLink` component in that case.

Reply:
I've updated the `ExternalReference` component to not to use the `ExternalLink` component when there is no link.